go/pkg/bertyprotocol/group_context.go

```diff
@@ -14,12 +14,12 @@ type groupContext struct {
 	group           *bertytypes.Group
 	metadataStore   *metadataStore
 	messageStore    *messageStore
-	messageKeystore *MessageKeystore
```

Comment:
the next time, I suggest you to make a quick PR just for this instead of keeping this small change in a long-running PR

Reply:
Done -> https://github.com/berty/berty/pull/2467